extension/war/src/main/resources/locale/portal/HamburgerMenu_en.properties

```diff
@@ -1,5 +1,5 @@
 menu.spaces.lastVisitedSpaces=Recent spaces
-menu.spaces.recentSpaces=Filter spaces
+menu.spaces.recentSpaces=Filter spaces here
```

Comment:
A change to make on crowdin as well @SaraBoutej @Samuel-Renault-eXo

Reply:
Thanks for the reminder, I have had validated it days ago but it seems it didn't work. So I did it again. To check in few days...